modules/nf-core/cellranger/mkfastq/main.nf

```diff
@@ -47,8 +48,8 @@ process CELLRANGER_MKFASTQ {
     }
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    mkdir -p "${prefix}_outs/outs/fastq_path/"
-    # data with something to avoid breaking nf-test java I/O stream
+    mkdir -p "${prefix}_outs/outs/fastq_path/sample/files/"
```

Comment:
Hey just stumbled on this. 

This isn't a stub test. The files shouldn't need to have anything in them, and you shouldn't need to run gzip or anything.

It should just create the expected output files, so they could be passed to another stub process.

This is more of a "mock" I think?